R/rmarkdown.R

```diff
@@ -37,6 +37,9 @@ render_rmarkdown <- function(pkg, input, output, ..., seed = NULL, copy_images =
       callr::r_safe(rmarkdown_render_with_seed, args = args, show = !quiet),
       error = function(cnd) {
         lines <- strsplit(gsub("^\r?\n", "", cnd$stderr), "\r?\n")[[1]]
+        lines <- lines[nchar(lines) >0]
+        lines <- gsub("{", "{{", lines, fixed = TRUE)
```

Comment:
Would you mind pulling this out into an `escape_cli()` function? I think that makes it slightly easier to understand the purpose of the code, and I suspect we'll need to use it elsewhere.

Reply:
Done! Thanks for your quick review, @hadley!